DOCUMENTATION.md

```diff
@@ -1045,7 +1058,7 @@ You can also set up [callbacks](#callbacks) for received text messages.
 
 ## Enable received text messages
 
-Contact the GOV.UK Notify team on the [support page](https://www.notifications.service.gov.uk/support) or through the [Slack channel](https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC) to enable receiving text messages for your service.
+Contact the Notify team using the [support page](https://www.notifications.service.gov.uk/support) or [chat to us on Slack](https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC) to request a unique number for text message replies.
```

Comment:
Just checking, we want to drop the `GOV.UK` from `GOV.UK Notify` in this case?

Reply:
Good spot. Whether or not we include the GOV.UK is about context.

There's a useful rule here: https://github.com/alphagov/notifications-manuals/wiki/A-to-Z-of-style#govuk-notify) but we need to tailor it for documentation.

I don't think it makes a _huge_ difference either way in this case – but adding the GOV.UK is probably the better choice.

This issue may exist in other PRs.

Reply:
Thanks for linking to the style guide to back this up. This issue does indeed exist for other PRs (appears to be in the slack link section and also in `To send Notify a request to go live`). I'll stop pointing it out in PR reviews as it is the same across the clients and leave you to decide if any changes are needed.